tests/tornado/test_tchannel.py

```diff
@@ -88,3 +89,23 @@ def test_advertise_should_listen_if_not_called_yet(tchannel, mock_server):
     )
 
     assert tchannel.is_listening() is True
+
+
+def test_advertise_should_take_a_router_file(tchannel, mock_server):
+
+    mock_server.expect_call('ad', 'json').and_write(
+        headers="",
+        body={"hello": "world"},
+    )
+
+    host_path = os.path.join(
```

Comment:
Reverted this change, since I tested a wrong tchannel objected.
The new change is in tests/test_tchannel.py